internal/backend/rclone/stdio_conn.go

```diff
@@ -28,21 +29,25 @@ func (s *StdioConn) Write(p []byte) (int, error) {
 	return n, err
 }
 
-// Close closes both streams.
+// Close closes the stream to the child process.
```

Comment:
Maybe this is the right time to rename stdin/stdout to toRclone/fromRclone. I was about to say that the comment is wrong, until I checked the code and realized I wanted to make that change myself and decided on the smaller patch.

Reply:
I've renamed the stream to send and receive